src/guc.c

```diff
@@ -546,6 +548,30 @@ _guc_init(void)
 							 NULL,
 							 NULL);
 
+	DefineCustomBoolVariable("timescaledb.partitionwise_plain_sorted_aggregation",
+							 "Enable partition-wise plain/sorted aggregation per chunk",
+							 "Enable the partition-wise aggregation pushdown to the"
+							 "chunk level",
+							 &ts_guc_enable_partitionwise_plain_sorted_aggregation,
+							 true,
+							 PGC_USERSET,
+							 0,
+							 NULL,
+							 NULL,
+							 NULL);
+
+	DefineCustomBoolVariable("timescaledb.partitionwise_hashed_aggregation",
+							 "Enable partition-wise hashed aggregation per chunk",
+							 "Enable the partition-wise aggregation pushdown to the"
+							 "chunk level",
+							 &ts_guc_enable_partitionwise_hashed_aggregation,
+							 true,
+							 PGC_USERSET,
+							 0,
+							 NULL,
+							 NULL,
+							 NULL);
+
```

Comment:
why do we need 2 gucs here?

Reply:
Those GUCs dont follow our usual naming for GUCs

Reply:
I was sure I wrote a comment about this as well, but it's not there... Probably one GUC + using the normal PG `enable_hashagg` to disable the hash aggregation specifically would be easier to use.

Reply:
The intention was to have the ability to force the creation of plain/sorted and hashed aggregation plans. 

Replaced by one GUC called `timescaledb.enable_partitionwise_aggregation` and use `enable_hashagg` as proposed by @akuzm .